mlxtend/regressor/stacking_cv_regression.py

```diff
@@ -0,0 +1,98 @@
+# Out-of-fold stacking regressor
+#
+# For explanation of approach, see:
+# dnc1994.com/2016/05/rank-10-percent-in-first-kaggle-competition-en/#Stacking
+#
+# Sebastian Raschka 2014-2017
+# mlxtend Machine Learning Library Extensions
+#
+# An ensemble-learning meta-regressor for out-of-fold stacking regression
+# Author: Eike Dehling <e.e.dehling@gmail.com>
+#
+# License: BSD 3 clause
+
+from sklearn.base import BaseEstimator
+from sklearn.base import RegressorMixin
+from sklearn.base import TransformerMixin
+from sklearn.base import clone
+from sklearn.model_selection import KFold
+from ..externals import six
+from ..externals.name_estimators import _name_estimators
+import numpy as np
+
+
+class StackingCVRegressor(BaseEstimator, RegressorMixin, TransformerMixin):
+    def __init__(self, regressors, meta_regressor, n_folds=5):
+        self.regressors = regressors
+        self.meta_regressor = meta_regressor
+        self.named_regressors = {key: value for
+                                 key, value in
+                                 _name_estimators(regressors)}
+        self.named_meta_regressor = {'meta-%s' % key: value for
+                                     key, value in
+                                     _name_estimators([meta_regressor])}
+        self.n_folds = n_folds
+
+    def fit(self, X, y):
+        self.regr_ = [list() for x in self.regressors]
+        self.meta_regr_ = clone(self.meta_regressor)
+
+        kfold = KFold(n_splits=self.n_folds, shuffle=True)
+
+        out_of_fold_predictions = np.zeros((X.shape[0], len(self.regressors)))
+
+        #
+        # The outer loop iterates over the base-regressors. Each regressor
+        # is trained and makes predictions, after which we train the
+        # meta-regressor on their combined results.
+        #
+        for i, clf in enumerate(self.regressors):
```

Comment:
Not that important since it's an internal variable, but I would suggest changing `clf` to `regr` or so